pennylane/transforms/add_noise.py

```diff
@@ -74,7 +78,14 @@ def add_noise(tape, noise_model, level=None):
         def noise2(op, **kwargs):
             qml.ThermalRelaxationError(op.parameters[0] * 0.5, kwargs["t1"],  kwargs["t2"], 0.6, op.wires)
 
-        noise_model = qml.NoiseModel({fcond1: noise1, fcond2: noise2}, t1=2.0, t2=0.2)
+        fcond3 = qml.noise.meas_eq(qml.expval) & qml.noise.wires_in([0, 1])
+        noise3 = qml.noise.partial_wires(qml.PhaseFlip, 0.2)
+
+        noise_model = qml.NoiseModel(
+            {fcond1: noise1, fcond2: noise2},
+            meas_map = {fcond3: noise3},
+            t1=2.0, t2=0.2
+        )
```

Comment:
I executed this example by changing RX with qml.adjoint(RX) to see what happened 
I got an unexpected error:

```python
@partial(qml.transforms.add_noise, noise_model=noise_model)
@qml.qnode(dev)
def circuit(w, x, y, z):
    qml.adjoint(qml.RX(w, wires=0))
    qml.RY(x, wires=1)
    qml.CNOT(wires=[0, 1])
    qml.RY(y, wires=0)
    qml.RX(z, wires=1)
    return qml.expval(qml.Z(0) @ qml.Z(1))
```

ValueError: pe must be between 0 and 1.

Reply:
You will need to change the definition of the `noise2` definition to use the absolute value of the parameters for the `RX`:

```py
def noise2(op, **kwargs):
    qml.ThermalRelaxationError(abs(op.parameters[0]) * 0.5, kwargs["t1"],  kwargs["t2"], 0.6, op.wires)
```